lib/client/interface.ts

```diff
@@ -46,7 +46,7 @@ export type TwitchMessageEvents = {
 
 // these are all other messages that are not mapped to twitch messages specifically, e.g. 001
 export interface IRCMessageEvents {
-  [command: string]: [IRCMessage];
+  [command: string & Record<never, never>]: [IRCMessage];
```

Comment:
`{}` is usually a banned type by [this linting rule](https://typescript-eslint.io/rules/ban-types) (disabled in this repo) i guess it could be used here without really affecting anything, i used a Record because i always get linting errors when i use `{}` in other codebases